.github/workflows/dependencies/dpcpp.sh

```diff
@@ -19,7 +19,7 @@ sudo apt-get update
 sudo apt-get install -y --no-install-recommends \
     build-essential \
     cmake           \
-    intel-oneapi-dpcpp-cpp-compiler intel-oneapi-mkl-devel \
+    intel-oneapi-dpcpp-cpp-compiler \
```

Comment:
Dang it, we need `oneapi/mkl/rng/device.hpp` which only comes in a huge `intel-oneapi-mkl-devel` package.